modules/ALFA/__init__.py

```diff
@@ -0,0 +1,4 @@
+# flake8: noqa
```

Comment:
Rather than running `flake8` and disabling all errors it would be better not to run the linter on this file at all.